File: articles/authentication-authorization-and-computational-identity/typescript/identity_access.ts
const weights: number[] = [0.10,0.11,0.11,0.09,0.09,0.10,0.09,0.09,0.08,0.06,0.06,0.02];
const score = weights.reduce((total, weight) => total + 0.75 * weight, 0) * 100;
console.log(score.toFixed(3));

File: articles/authentication-authorization-and-computational-identity/c/identity_access.c
#include <stdio.h>

int main(void) {
    double weights[] = {0.10,0.11,0.11,0.09,0.09,0.10,0.09,0.09,0.08,0.06,0.06,0.02};
    double score = 0.0;
    for (int i = 0; i < 12; ++i) score += 0.75 * weights[i];
    printf("%.3f\n", score * 100.0);
    return 0;
}
